uint16 mode 
float64 n
float64 e
float64 d
float64 yaw
float64 PTZ_pitch
uint16 fly_time